postgres/check.py

```diff
@@ -716,6 +716,8 @@ def _get_custom_metrics(self, custom_metrics, key):
     def check(self, instance):
         host = instance.get('host', '')
         port = instance.get('port', '')
+        if port != '':
+            port = int(port)
```

Comment:
Specifically, the role provided by datadog (https://github.com/DataDog/ansible-datadog) uses a templating feature of jinja2 to render out the final yaml for the check configuration, and this specific piece of code unfortunately puts quotes around the port number. The upstream ansible project refers to this here: https://github.com/ansible/ansible/issues/9362
So yes, it's a workaround, but one with no harm I believe.